requirements-dev.txt

```diff
@@ -4,6 +4,7 @@ Appium-Python-Client<=0.52
 flake8==3.8.3
 nose==1.3.7
 PyYAML>=1.0
+splinter<=0.16.0
```

Comment:
which plugin is installing this?

Reply:
It doesn't come from a plugin. It comes from 'behave'.